docs/guides/cloud/debugging/test-replay.mdx

```diff
@@ -0,0 +1,136 @@
+---
+title: Test Replay
+sidebar_position: 20
+---
+
+:::info
+
+## <Icon name="graduation-cap" /> What you'll learn
+
+- What Test Replay is and why you should use it
+- How to get started with Test Replay in Cypress Cloud
+- How to debug failed and flaky CI test runs more accurately and efficiently
+
+:::
+
```

Comment:
should probably put an info tip around here to let users know its cy13 and up only